src/main/java/xyz/deseteral/lebkuchenfm/domain/dto/GenericCommandResponseDto.java

```diff
@@ -0,0 +1,17 @@
+package xyz.deseteral.lebkuchenfm.domain.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class GenericCommandResponseDto {
```

Comment:
Jest to obiekt zwracany na API, to może tam powinien się właśnie znajdować? I wtedy mógłby być finalny i package-scope?

Reply:
Fixed :)